src/Features/LanguageServer/Protocol/Handler/Definitions/AbstractGoToDefinitionHandler.cs

```diff
@@ -119,6 +119,19 @@ static bool ShouldInclude(INavigableItem item, bool typeOnly)
                         return false;
                 }
             }
+
+            static async Task<IEnumerable<INavigableItem>?> GetDefinitions(Document document, int position, CancellationToken cancellationToken)
+            {
+                var findDefinitionService = document.Project.LanguageServices.GetService<IFindDefinitionService>();
```

Comment:
```suggestion
                var findDefinitionService = document.GetLanguageService<IFindDefinitionService>();
```

Reply:
in general, this is also worth a comment.  i.e.: try the new interface first and see if we can get results from it.  If not, fallback to the legacy interface.  Once http:// is addressed and all languages are on the new interface, we can remove hte old interface.